Makefile

```diff
@@ -0,0 +1,29 @@
+# Required for merging:
+
+pip:
+	pip install -r requirements.txt
+	pip install -e .
+
+build-and-test: build black mypy test
+
+build:
+	python -m scripts.generate_init --check
+
+black:
+	black --check .
+
+mypy:
+	mypy pyteal
+
+test:
+	pytest
+
+
+# Extras:
+
+coverage:
+	pytest --cov-report html --cov=pyteal
+
+blackbox:
+	HAS_ALGOD=TRUE pytest tests/semantic_test.py
```

Comment:
@tzaffi I feel like there's a chance to unify naming between this PR and https://github.com/algorand/py-algorand-sdk/pull/298.  To illustrate what I'm seeing, I opened an optional PR for your review: https://github.com/algorand/pyteal/pull/259.

Let's resolve the thread once the child PR is either closed or merged.

Reply:
@tzaffi Thanks for being willing to take #259 in - Resolving.  ☕

Reply:
🙏